### ISSUE TITLE
Support encoding / decoding binary fields

### ISSUE THREAD

**Issue by mdwaud** [datafusion-contrib/datafusion-postgres#27]
When attempting to return binary data I'm getting `Unsupported Datatype Binary and array BinaryArray`. It looks like except for `into_pg_type` binaries aren't really implemented yet.

**Comment by sunng87** [comment]
Yes we have a few types to be covered in the conversion module. And I'm thinking about extracting a new crate for conversion between postgres type and arrow.

**Comment by mdwaud** [comment]
I gave a quick try of the latest release and it looks good so far! Will report back if I encounter any issues.

**Comment by sunng87** [comment]
The new version just brings initial support for binary field. But with our experience in GreptimeDB, there are some more binary serieliazation options to support for full postgres compatibility.
